### PR TITLE
Provide const assertion on tokens in TypeScript

### PR DESCRIPTION
**Pull request by Peeja**
In the past, we've gotten a lot of value out of typing our token values `as const`. This means that in VS Code, for instance, the Intellisense will show you the values alongside the token names. Ideally, engineers never need to know the values when they're writing code, because they have the token name from the designer, but we've found that from time to time folks still have to look up a token by its value. We're using Figmagic to import our tokens into a shared library, and then using that library in multiple apps, so looking up a token name by value in the code means digging into the shared library's repo, while this change makes it possible in the editor where you're already working.

### Before (no values available in Intellisense)

image

### After (values available in Intellisense)

image

### CODE REVIEW

File: bin/frameworks/filesystem/getFileContentAndPath.ts [hunk 1]
```diff
@@ -132,14 +132,16 @@ const getTokenEnumString = (file: string | ProcessedToken, name: string, format:
  * @description Get file data string for tokens using null/no data type
  */
 const getTokenString = (file: string | ProcessedToken, name: string, format: string) => {
+  console.log(format);
```
Comment (mikaelvesavuori) [inline bin/frameworks/filesystem/getFileContentAndPath.ts:135]: Can you remove this? Other than that, it all looks good!

**Review by mikaelvesavuori — CHANGES_REQUESTED** [review]
Just a single comment: if you could please remove the console.log() then I am happy to merge this! :)

**Comment by mikaelvesavuori** [comment]
Also: A very good addition! Thanks for contributing. Upon merging, I will add your name to the contributors list if you don't mind?

File: bin/frameworks/filesystem/getFileContentAndPath.ts (hunk 1 above)
Comment (Peeja) [reply bin/frameworks/filesystem/getFileContentAndPath.ts:135]: 🤦🏻‍♀️ Good catch, thank you! 😛

Comment (Peeja) [reply bin/frameworks/filesystem/getFileContentAndPath.ts]: Removed!

**Comment by Peeja** [comment]
Happy to be on the list, thanks!

**Comment by mikaelvesavuori** [comment]
Looks good! Merging.

**Pull request merged**